bioblend/_tests/TestGalaxyJobs.py

```diff
@@ -23,16 +34,67 @@ def test_run_and_rerun_random_lines(self):
         }
 
         original_output = self.gi.tools.run_tool(
-            history_id=history_id,
+            history_id=self.history_id,
             tool_id="random_lines1",
             tool_inputs=tool_inputs,
             input_format='21.01'
         )
         original_job_id = original_output['jobs'][0]['id']
+        self._wait_job(original_job_id)
```

Comment:
This can be postponed a bit, I think.
```suggestion
```

Reply:
I think that `_build_for_rerun` on the next line won't work properly without this. I added the rest of your changes.

Reply:
I don't think the job needs to be completed to be re-runnable?